cmd/juju/status/status.go

```diff
@@ -172,11 +171,7 @@ func (c *statusCommand) SetFlags(f *gnuflag.FlagSet) {
 
 	f.IntVar(&c.retryCount, "retry-count", 3, "Number of times to retry API failures")
 	f.DurationVar(&c.retryDelay, "retry-delay", 100*time.Millisecond, "Time to wait between retry attempts")
-
-	if runtime.GOOS != "windows" {
-		// The watch flag is only available for unix-based systems.
-		f.DurationVar(&c.watch, "watch", 0, "Watch the status every period of time")
-	}
+	f.DurationVar(&c.watch, "watch", 0, "Watch the status every period of time")
```

Comment:
A reference to requiring Viddy here, perhaps? This string is used to generate `juju status`'s help string

Reply:
Thx, for your comment.  Fixed this issue.